### ISSUE TITLE
syntax for HETS API extensions

### ISSUE THREAD

**Issue by mcodescu** [spechub/Hets#1561]
* the (full) theory of a node in a DG:
```GET /dg/coded-iri/command-list?options&node=iri ```
* the translation of the (full) theory of a node in a DG along a comorphism:
```GET /dg/coded-iri/command-list?options&node=iri&translation=iri ```
* the translation of a graph along a comorphism
```GET /dg/coded-iri/command-list?options&translation=iri ```

**Comment by tillmo** [comment]
sounds good! A smooth extension of what is already there.

**Comment by cmaeder** [comment]
The leading "dg" was supposed to indicate that you would get a development graph, so maybe "theory" should be used in the first two cases (that may require `node=iri` to be in the query part or would only output the theory  of the first or a solitary node).

**Comment by tillmo** [comment]
Agreed. A translation of the (full) theory of a node in a DG along a composition of comorphisms would be written `GET /theory/coded-iri/command-list?options&node=iri&translation=iri1;...;irin`
@clange, is the semicolon legal for separating IRIs?

**Comment by clange** [comment]
According to RFC 3986 the semicolon is legal. However, for this same reason, it might occur in any of the `iri1` to `irin`. If it occurs there it will have to be escaped as `%3B`, which shouldn't be a problem.

FYI, IIRC Florian Rabe makes a lot of use of `;` in his MMT API.

**Comment by eugenk** [comment]
`&` and `;` in a query string of an IRI have the same semantics/are interchangeable. We should not redefine the semantics of the `;` in the Hets API. Maybe a better solution would be to use another HTTP-verb than `GET` and supply the options in the request body.

**Comment by eugenk** [comment]
... or simply to use a `,` like in the command line:
```
 ❯ hets
supply option -G or -g and/or file arguments
Usage: hets [OPTION ...] [FILE ...] [+RTS -?]
[...]
  -t TRANS      --translation=TRANS     translation option
                                          is a colon-separated list
                                          of one or more from: SIMPLE-ID
```

**Comment by tillmo** [comment]
Good, a comma is a good idea.
